### DISCUSSION TITLE
Will there be a 240fps mode?

### DISCUSSION THREAD

**Discussion by Berdasco99** [WakuWakuPadoru/WuWa_Simple_FPSUnlocker#4] [Q&A] [answered]
Just curious if that will come in the future, i have a 240hz display so it'd be pretty cool to use it's full capabilities, regardless great mod thanks!!

**Comment by WakuWakuPadoru** [comment, marked as answer]
> Just curious if that will come in the future, i have a 240hz display so it'd be pretty cool to use it's full capabilities, regardless great mod thanks!!

No. The game strictly enforces minimum and maximum FPS values. So while the Unlocker can technically set 240 in the settings file and it will work at the start, the moment that you try to login, it automatically drops it down to 120 FPS.

That's why I limited the FPS Value in the unlocker to 120 FPS, so as not to give anyone false hope that anything above that 120 is actually supported by the game.

**Reply by Stickpage2020** [reply]
but why is there a fps unlocker that's possible at doing 144 hrts and more for genshin impact?

**Reply by WakuWakuPadoru** [reply]
> but why is there a fps unlocker that's possible at doing 144 hrts and more for genshin impact? 

Genshin is a different game and runs on a different engine, so different methods are used. Unlockers for Genshin modifies how the game reads from memory which is a lot more risky as that's pretty similar to how some hacks and injectors work. 

This unlocker modifies the configuration settings and so, is a lot safer. If you set any number below 25 or above 120, WuWa automatically adjusts it back to 25 or 120 when you login to your Rover.

In other words, WuWa doesn't allow you to go below 25 or above 120. Unlockers cannot surpass this limit.
